owncloudApp/src/androidTest/java/com/owncloud/android/settings/security/SettingsSecurityFragmentTest.kt

```diff
@@ -425,6 +426,34 @@ class SettingsSecurityFragmentTest {
         assertFalse(prefTouchesWithOtherVisibleWindows.isChecked)
     }
 
+    @Test
+    fun passcodeLockNotVisible() {
+        every { securityViewModel.isSecurityEnforcedEnabled() } returns true
+        launchTest()
+        assertFalse(prefPasscode.isVisible)
+    }
+
+    @Test
+    fun patternLockNotVisible() {
+        every { securityViewModel.isSecurityEnforcedEnabled() } returns true
+        launchTest()
+        assertFalse(prefPattern.isVisible)
+    }
+
+    @Test
+    fun passcodeLockVisible() {
+        every { securityViewModel.isSecurityEnforcedEnabled() } returns false
+        launchTest()
+        assertTrue(prefPasscode.isVisible)
+    }
+
+    @Test
+    fun patternLockVisible() {
+        every { securityViewModel.isSecurityEnforcedEnabled() } returns false
+        launchTest()
+        assertTrue(prefPattern.isVisible)
+    }
```

Comment:
Great that you decided to add UI tests! 👏  But in these 2 tests, the `every` clauses are not necessary as you already set that this mocked method returns `false` in `setUp()`. In the other 2 tests you added, it IS necessary, as the value changes from what you specified at the beginning 😀

Reply:
Good catch! Thanks 😉